src/types.js

```diff
@@ -24,15 +24,16 @@ export type PayPalEligibility = {|
 |};
 
 export type FundingProductEligibility = {|
-    eligible? : boolean
+    eligible? : boolean,
+    variant : string
```

Comment:
Should this be optional?

Reply:
Good question, this `variant` key will be null by default and in special use cases return a string. 

I think if I add it as optional it will error out for that `null` use case. 
https://flow.org/en/docs/types/objects/#toc-optional-object-type-properties

Is there a better way to type this?

Reply:
I see.  I think you can do `variant? : ?string`.  I believe this mean its optional and can be null or undefined.

Reply:
Makes sense, I updated. Thanks!